src/Vehicle/InitialConnectStateMachine.cc

```diff
@@ -216,6 +216,10 @@ void InitialConnectStateMachine::_stateRequestProtocolVersion(StateMachine* stat
         if (sharedLink->linkConfiguration()->isHighLatency() || sharedLink->isPX4Flow() || sharedLink->isLogReplay()) {
             qCDebug(InitialConnectStateMachineLog) << "Skipping REQUEST_MESSAGE:PROTOCOL_VERSION request due to link type";
             connectMachine->advance();
+        }
```

Comment:
```suggestion
        } else if (vehicle->apmFirmware()) {
             qCDebug(InitialConnectStateMachineLog) << "Skipping REQUEST_MESSAGE:PROTOCOL_VERSION request due to Ardupilot firmware";
             connectMachine->advance();
         } else {
```

Reply:
Can you just append the "else if" to line 219?

Reply:
Done.